lab-jason/test/arithmetic-test.js

```diff
@@ -0,0 +1,28 @@
+const assert = require('assert');
+const {expect} = require('chai');
+const arithmetic = require('../lib/arithmetic');
+
+describe('arithmetic', function(){
+  describe('#add', function(){
+    it('should return an number', function (){
+      expect(arithmetic.add(10,1)).to.be.a('number');
+    });
+    it('should return null if params are NaN', function(){
+      assert.strictEqual(arithmetic.add('cat',7), null);
+    });
+    it('should math correctly', function(){
+      assert.strictEqual(arithmetic.add(1,2), 3);
+    });
+  });
+  describe('#sub', function(){
+    it('should return an number', function (){
+      expect(arithmetic.sub(10,1)).to.be.a('number');
+    });
+    it('should return null if params are NaN', function(){
+      assert.strictEqual(arithmetic.sub('cat',7), null);
```

Comment:
Based on the test name, I was expecting to see NaN as an actual parameter here. Not necessary now, but as discussed in class this is the sort of test where multiple `assert`s for different invalid inputs would be ideal.